ksqldb-parser/src/test/java/io/confluent/ksql/util/QueryMaskTest.java

```diff
@@ -0,0 +1,171 @@
+        + "    \"mode\"='bulk',\n"
+        + "    \"topic.prefix\"='jdbc-',\n"
+        + "    \"table.whitelist\"='users',\n"
+        + "    \"key\"='username');";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+        final String expected = "CREATE SINK CONNECTOR `test-connector` WITH "
+            + "(\"connector.class\"='PostgresSource', "
+            + "'connection.url'='[string]', "
+            + "\"mode\"='[string]', "
+            + "\"topic.prefix\"='[string]', "
+            + "\"table.whitelist\"='[string]', "
+            + "\"key\"='[string]');";
+
+    assertThat(maskedQuery, is(expected));
+  }
+
+  @Test
+  public void shouldMaskInvalidSinkConnector() {
+    // Given:
+    // Typo in "CONNECTOR" => "CONNECTO"
+    final String query = "CREATE Sink CONNECTO `test-connector` WITH ("
+        + "    \"connector.class\" = 'PostgresSource', \n"
+        + "    'connection.url' = 'jdbc:postgresql://localhost:5432/my.db',\n"
+        + "    \"mode\"='bulk',\n"
+        + "    \"topic.prefix\"='jdbc-',\n"
+        + "    \"table.whitelist\"='users',\n"
+        + "    \"key\"='username');";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+    final String expected = "CREATE Sink CONNECTO `test-connector` WITH ("
+        + "    \"connector.class\" = 'PostgresSource', \n"
+        + "    'connection.url'='[string]',\n"
+        + "    \"mode\"='[string]',\n"
+        + "    \"topic.prefix\"='[string]',\n"
+        + "    \"table.whitelist\"='[string]',\n"
+        + "    \"key\"='[string]');";
+
+    assertThat(maskedQuery, is(expected));
+  }
+
+  @Test
+  public void shouldNotMaskInvalidCreateStreamWithoutQuote() {
+    // Given:
+    // Typo in "WITH" => "WIT"
+    final String query = "CREATE STREAM `stream` WIT ("
+        + "    format = 'avro', \n"
+        + "    kafka_topic = 'test_topic',"
+        + "\"partitions\"=   3,\n"
+        + ");";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+    assertThat(maskedQuery, is(query));
+  }
+
+  @Test
+  public void shouldNotMaskValidCreateStream() {
+    // Given:
+    final String query = "CREATE STREAM `stream` (id varchar) WITH (format = 'avro', kafka_topic = 'test_topic', partitions=3);";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+    assertThat(maskedQuery, is(query));
+  }
+
+  @Test
+  public void shouldMaskInvalidCreateStreamWithQuotes() {
+    // Given:
+    // Typo in "WITH" => "WIT"
+    final String query = "CREATE STREAM `stream` (id varchar) WIT ('format' = 'avro', \"kafka_topic\" = 'test_topic', partitions=3);";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+    final String expected  = "CREATE STREAM `stream` (id varchar) WIT ('format'='[string]', \"kafka_topic\"='[string]', partitions=3);";
```

Comment:
Add a quick comment into this test case for why this is expected (that the CREATE STREAM has properties replaced)? I could see a future reader easily having this question.